### DISCUSSION TITLE
How should I use theses options the docker compose ?

### DISCUSSION THREAD

**Discussion by Morgan-6Freedom** [photoprism/photoprism#3397] [Q&A]
Hello.
In the [documentation to install the docker compose](https://docs.photoprism.app/getting-started/docker-compose/), it says to change "PHOTOPRISM_ADMIN_PASSWORD" and "volumes" but nothing else.
However there are multiples comments that explains multiple things in this file, but not explained in the [documentation](https://docs.photoprism.app/getting-started/docker-compose/).

What should I do with :

```
## Run/install on first startup (options: update https gpu tensorflow davfs clitools clean):
# PHOTOPRISM_INIT: "https gpu tensorflow"
## Hardware Video Transcoding:
# PHOTOPRISM_FFMPEG_ENCODER: "software"        # FFmpeg encoder ("software", "intel", "nvidia", "apple", "raspberry")
# PHOTOPRISM_FFMPEG_BITRATE: "32"              # FFmpeg encoding bitrate limit in Mbit/s (default: 50)
## Run as a non-root user after initialization (supported: 0, 33, 50-99, 500-600, and 900-1200):
# PHOTOPRISM_UID: 1000
# PHOTOPRISM_GID: 1000
# PHOTOPRISM_UMASK: 0000
## Start as non-root user before initialization (supported: 0, 33, 50-99, 500-600, and 900-1200):
# user: "1000:1000"
```

**Comment by graciousgrey** [comment]
Hey 👋 

You can find more information on config options [here](https://docs.photoprism.app/getting-started/config-options/) and on video transcoding [here](https://docs.photoprism.app/getting-started/advanced/transcoding/)

**Comment by Morgan-6Freedom** [comment]
So 
`# PHOTOPRISM_INIT: "https gpu tensorflow"`
should be uncommented for first launch ?
If yes, could you update the documentation ?
if yes, I already started my photoprism, can I launched it afterward ?

**Reply by lastzero** [reply]
This is optional and increases the startup time due to the additional packages that are getting installed.

**Reply by Morgan-6Freedom** [reply]
What is the goal to uncomment it ?
if it increase startup time for first launch then I comment it again, it's ok ?
Do I need this to have object or face recognition ?

**Reply by lastzero** [reply]
See our hardware transcoding documentation. It's described there. If you don't need additional drivers, you don't need to touch it. You won't lose any standard functionality.
